contributing.md

```diff
@@ -4,26 +4,27 @@ Contributing
 ## Issue Tracking
 
 New feature requests, changes, enhancements, non-methodology features, and bug reports can be filed as new issues in the
-[Github.com issue tracker](https://github.com/IEA-Task-43/digital_wra_data_standard/issues) at any time. Please be sure to fully describe the
+[GitHub.com issue tracker](https://github.com/IEA-Task-43/digital_wra_data_standard/issues) at any time. Please be sure to fully describe the
 issue.
 
 <!---For other issues, please email the Task 43 distribution list at `xx`.--->
 
 ## Repository
 
-The digital WRA digital_wra_data_standard repository is hosted on Github, and located here: https://github.com/IEA-Task-43/digital_wra_data_standard
+The digital WRA digital_wra_data_standard repository is hosted on GitHub, and located here: https://github.com/IEA-Task-43/digital_wra_data_standard
 
 This repository is organized using a modified git-flow system. Branches are organized as follows:
 
 - master: Stable release version. Must have good test coverage and may not have all the newest features.
 - dev: Development branch which contains the newest features. Tests must pass, but code may be unstable.
-- feature/xxx: Branch from dev, should reference a github issue number.
+- feature/xxx: Branch from dev, should reference a GitHub issue number.
 
-To work on a feature, please fork digital_wra_data_standard first and then create a feature branch in your own fork.
-Work out of this feature branch before submitting a pull request.
-Be sure to periodically synchronize the upstream dev branch into your feature branch to avoid conflicts in the pull request.
+To work on a feature, please fork digital_wra_data_standard first. Then clone your fork locally when you are ready to get to work. If your git status is unclean after you clone  (e.g., lots of untracked files and changes), chances are you are dealing with a Unicode normalization issue (specifically, the Unicode decomposition of filenames under macOS). This can be an issue on project with contributors using a mix of MacOS, Linux, and Windows. To resolve it, delete your clone. Then issue `git config --global core.precomposeunicode true` and clone your fork again. 
```

Comment:
Which branch from digital_wra_data_standard do they fork? I am assuming the Dev branch (when we make one)?

Reply:
Yes, @stephenholleran, that is my understanding of @Dynorat 's original intent. I'm happy to make that more explicit.

Reply:
If you could that would be great. Thanks.

Reply:
I added a second commit. The change was to name the source branch (dev) for the feature branch explicitly. More clear now?

Reply:
Brilliant! I forgot that it was such a small change I could have done it myself.